### ISSUE TITLE
Startup writes a file to /dev/null on the Installation Drive

### ISSUE THREAD

**Issue by PetePete1984** [PoE-TradeMacro/POE-TradeMacro#817]
Windows version: 10 64-Bit

I noticed a recently changed file in my E:\Dev folder, called "null", 3KB in size. It contained the JSON result of a PoE API "leagues" request, basically.
The only thing that would be remotely interested in reading this API would be the Trade Macro, which is also installed on the E drive.
Putting one and one together and checking the source I've found the culprit: https://github.com/PoE-TradeMacro/POE-TradeMacro/blob/676ff0c6169c0ce537da6421c74f6b16655badde/lib/PoEScripts_Download.ahk#L84

So basically, what works on Linux to just pipe the result of a request into /dev/null nirvana will create a new file on Windows if you happen to have a Dev folder on the same drive as your TradeMacro installation.

Minor nuisance at worst, but still surprising.

https://stackoverflow.com/questions/32488162/curl-suppress-response-body suggests using the "-o nul" switch for Windows, but that would require checking the OS in the first place (although I admittedly don't know if AHK even runs on Linux)

**Comment by Eruyome** [comment]
While AHK may run under linux via Wine or similiar software I know that TradeMacro doesn't run on linux "out of the box", and I'll never support linux anyway, that's asking too much. 

I've changed the switch in my dev version and will see if there are any issues.
